deploy.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script is to be ansibilized
+# run as root or with sudo
+
+# create directories
+mkdir -p /var/lib/jirasync
+mkdir -p /var/run/jirasync
+mkdir -p /etc/jirasync 
+
+# copy config files
+cp config_sample.yaml /etc/jirasync/config.yaml
```

Comment:
We should add a conditional statement here to not copy config_sample.yaml file all the time to /etc/jirasync/config.yaml location.

Whenever we run deploy.sh second time it updates existing config.file

Reply:
```
#!/bin/bash
file="/etc/jirasync/config.yaml"
if [ ! -f "$file" ]
then
    cp config_sample.yaml /etc/jirasync/config.yaml
fi
```

Reply:
ok, I'll update, although this script is meant to run **once** because of the virtualenv creation etc..

To make really idempotent we must turn it in to Ansible play, also we can build a docker image.